timeseries/src/autogluon/timeseries/models/local/abstract_local_model.py

```diff
@@ -105,19 +89,35 @@ def preprocess(
             data = data.fill_missing_values()
         return data, known_covariates
 
+    def _get_default_hyperparameters(self) -> dict:
+        return {
+            "n_jobs": AG_DEFAULT_N_JOBS,
+            "use_fallback_model": True,
+            "max_ts_length": self.default_max_ts_length,
+        }
+
     def _fit(self, train_data: TimeSeriesDataFrame, time_limit: Optional[int] = None, **kwargs):
         self._check_fit_params()
 
         if time_limit is not None and time_limit < self.init_time_in_seconds:
             raise TimeLimitExceeded
 
-        # Initialize parameters passed to each local model
-        raw_local_model_args = self.get_hyperparameters().copy()
+        hyperparameters = self.get_hyperparameters()
+        # TODO: Replace with 'num_cpus' argument passed to fit (after predictor API is changed)
```

Comment:
Similar to the above, can these go inside `self.get_hyperparameters` I wonder

Reply:
I would prefer to perform the "preprocessing" of `_local_model_args` inside `_fit()` instead of moving it to `get_hyperparameters()`. If we do move it to `get_hyperparameters()`, the warning 
```
                f"{self.name} ignores following hyperparameters: {unused_local_model_args}. "
                f"See the docstring of {self.name} for the list of supported hyperparameters."
```
will be logged each time the user predicts with the model (e.g., during `leaderboard` or `evaluate`). I think that's unnecessary and the warning should only be logged once during `predictor.fit()`.